homeassistant/const.py

```diff
@@ -1051,6 +1051,58 @@ class UnitOfDataRate(StrEnum):
 """Deprecated: please use UnitOfDataRate.GIBIBYTES_PER_SECOND"""
 
 
+UNITS_OF_MEASUREMENT: Final[frozenset[str]] = frozenset(
+    {
+        unit
+        for unit_type in (
+            UnitOfApparentPower,
```

Comment:
Instead of creating this list manually, we could have a base class or something similar for all units and create the list dynamically.

Reply:
Discussed it internally, keep it a manual list